### DISCUSSION TITLE
Welcome to GeoPyTool Discussions!

### DISCUSSION THREAD

**Discussion by cycleuser** [GeoPyTool/GeoPyTool#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ringosta** [comment]
![Screenshot (5)](https://user-images.githubusercontent.com/92560210/137424645-23411609-9a96-4f1f-a558-6f37240ed707.png)
see the picture attached. what is the problem?. i cant conducting Trace and REE diagram. fyi, i only have 4 data and not all of row were filled.

**Reply by cycleuser** [reply]
This might be a bug, would you mind to tell me the function giving this Warning and even better to share a sample of your modified data file with me for further test?

**Reply by ringosta** [reply]
i dont know, but i will share my modified data. i hope the program will better soon. what is your email?. thanks.

**Reply by cycleuser** [reply]
The problem seems like caused by the "b.d" and "bd" inside the data sheet, and the "Alpha" shoud be form 0 to 1 as 1 means no transpearncy and 0 means no color.
After deleting these "b.d" and "bd" and set all "Alpha" to 0.6, it work out the plot of REE and Trace elements as following pictures.
The modified data file "Geochemistry1_Modified.xlsx" is attached in the email sent to you.

**Reply by cycleuser** [reply]
The NAN seems like to be caused by the different behavior of Pandas, used to import data sheet, on Windows it generate a lot of NAN, which didn't happen on Linux or macOS.

Thank you very much to let me know this!

At the moment, the data sheet should be CSV coded with UTF-8 without BOM. 
I gonna push a new version using CSV like this as default format.

**Comment by XShaeeee** [comment]
<img width="1177" alt="截屏2021-10-30 下午1 27 04" src="https://user-images.githubusercontent.com/93415416/139521502-42a773d4-16b8-4799-8b23-a644849cebb5.png">
请问一下 为什么MacOS字体发虚呢

**Reply by cycleuser** [reply]
我这设备有限，手上的几台老机器没能再现此情况，原因如何就不能确定了，可能是pyqt5在macOS下的字体显示如此？毫无头绪。
